themes/scss/widgets/text-elements.scss

```diff
@@ -6,10 +6,9 @@
 
 .Widget-textBigger { // 16 size
   @include text-overflow();
+  color: #2E3C43;
```

Comment:
Mmmm… I didn't add that, just moved it to the top so it passed our CSS rules… I'll put it to the light  theme.